handout/features/README.md

```diff
@@ -3,7 +3,7 @@
 ![ES6/TypeScript](../images/es6-typescript.png "ES6 and TypeScript")
 
 The language we usually call "JavaScript" is formally known as "EcmaScript".
-The new version of JavaScript, "EcmaScript 6" or "ES6", offers a number of new features that extend the power of the language.
-ES6 is not widely supported in today's browsers, so it needs to be transpiled to ES5. 
-You can choose between several transpilers, but we'll be using TypeScript, 
-which is what the Angular team uses to write Angular 2. Angular 2 makes use of a number of features of ES6 and TypeScript.
+The new version of JavaScript, "ECMAScript 2015" previously known as "ES6", offers a number of new features that extend the power of the language.
```

Comment:
As we spoke making global update to use ES6 for now, mention ES2015 being the same thing for now.